app/controllers/product_controller.rb

```diff
@@ -3,4 +3,12 @@ class SlowFoodApp
   get '/' do
     erb :welcome
   end
+
+
+
+  get '/menu' do
+    @menu = Product.all
```

Comment:
If you are storing a collection of products, then I would NOT call the variable `@menu` I woud do 
```
@menu = Menu.first 
# or identify the menu by an id
get '/menu/:id' do 
  @menu = Menu.find(params[:id]
end
```
Then, in your view (IF the association is set up properly) you'll have `@menu.products` available as a collection